packages/cli/src/commands/integration/client.ts

```diff
@@ -1,104 +0,0 @@
-import type Client from '../../util/client';
```

Comment:
This file has been split into its individual functions and moved to the `packages/cli/util/integration/` space.